beep/protocol/maccor_to_biologic_mb.py

```diff
@@ -1271,3 +1645,61 @@ def set_global_fields(seqs):
     with open(fp, "wb") as f:
         f.write(file_str.encode("ISO-8859-1"))
         print("created {}".format(fp))
+
+
+def add_cycle_nums_to_csvs(csv_and_transition_rule_file_paths):
```

Comment:
Is there a reason to keep this as a single argument instead of splitting out each of the components?

Reply:
Not particularly, we can change it if you prefer.

Reply:
If it's not too much of a pain, that might help clarify how to use the function. Alternatively, a small doc string that has an example input so that I can see what kinds of arguments are needed (absolute path, list of paths, etc.)

Reply:
I split the arg into 3, added some doc strings. Please let me know if they're unclear.